[04] View & Modifiers/[01] Challenge 1/WeSplit/Extensions/String+Ext.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  String+Ext.swift
+//  WeSplit
+//
+//  Created by Joe Pham on 2022-07-12.
+//
+
+import Foundation
+
+extension String {
+	
+	/// Generate localized version of the string, assuming the project has been set up to support multiple localizations
+	func localized() -> String {
+		return NSLocalizedString(
+			self,
+			tableName: "Localizable",
+			bundle: .main,
+			value: self,
+			comment: self
+		)
+	}
```

Comment:
Can also utilize the built-in API in iOS 15 to return a LocalizedStringKey, so that the compiler looks for a correspondent translation in your Localizable.strings automatically